src/meta/test/JobManagerTest.cpp

```diff
@@ -66,7 +66,7 @@ class JobManagerTest : public ::testing::Test {
           }
           delete p;
         });
-    jobMgr->status_ = JobManager::JbmgrStatus::NOT_START;
+    jobMgr->status_.store(JobManager::JbmgrStatus::NOT_START, std::memory_order_release);
```

Comment:
IIRC, it won't help to solve the problem that the job starts running before we set job manager to stop? Correct me if I'm wrong. `memory_order_release` is weaker than `memory_order_cst`

Reply:
Here use  `A = a ` to assign a value to the atomic variable, which is equivalent to` A.store(a)`,
store function  defaults to `std::memory_order_seq_cst`

Whether memory_order_seq_cst or memory_order_release is used here has no effect.

Reply:
Neither the `sc` model nor the `acquire-release `model can help to solve the problem that the job starts running before we set job manager to stop.

Reply:
So job could still be executed automatically before we set it to STOPPED, right?

Reply:
Yes.
Before setting
`jobMgr->status_ = JobManager::JbmgrStatus::STOPPED;`
 it will enter `JobManager::scheduleThread` to execute job.

However, when we use one `getJobManager`,  there is no problem.